docs/static/keystore.asciidoc

```diff
@@ -12,15 +12,30 @@ place of the secret value when you configure sensitive settings.
 The syntax for referencing keys is identical to the syntax for
 <<environment-variables, environment variables>>:
 
+[source,txt]
+-----
 `${KEY}`
```

Comment:
Do we want the `backticks` around ${KEY} preserved here? I think they were previously used for emphasis/presentation, rather than literal backticks, which is how they are presented in the `[source,txt]` block